src/services/firebase.js

```diff
@@ -14,21 +14,24 @@
 
 import admin from 'firebase-admin';
 import path from 'path';
+import fs from 'fs';
 
-let configFile = path.resolve(__dirname, '../data/firebase-admin-key.json');
+export default function initializeApp() {
+  if (admin.apps.length == 0) {
```

Comment:
As discussed, we've used `===` elsewhere, though I can't find this in the Google style guide.